src/cpp/src/UTIL/CheckCollections.cc

```diff
@@ -33,8 +31,31 @@ namespace UTIL{
 	if( it == _map.end() ){
 
 	  auto col = evt->getCollection( name ) ;
-
-	  const auto[ itx, inserted] = _map.emplace( name,  std::make_pair( col->getTypeName() , 0 )  )   ;
+    /* If the type of a collection is LCRelation we use want to read the entire collections instead of just the header to get the 'toType' and 'fromType'.  
+    * setReadCollectionNames({name}) allows reading of onlz certain colloctions by name instead of an entire event.
+    * This flag has to be unset after reading in order for the reading of the headers to function.
+    */
```

Comment:
```suggestion
    /* If the type of a collection is LCRelation we want to read the entire collections instead of just the header to get the 'ToType' and 'FromType'.  
    * setReadCollectionNames({name}) allows reading of only certain collections by name instead of an entire event.
    * This flag has to be unset after reading in order for the reading of the headers to function properly.
    */
```
Minor typos and clarifications